drivers/storage/rbd/utils/utils.go

```diff
@@ -0,0 +1,287 @@
+// +build !libstorage_storage_driver libstorage_storage_driver_rbd
```

Comment:
Hi @codenrhoden,

I might add a sub-package called `rbd-utils` and make the file `rbd-utils.go` use the package main, thereby creating a new CLI. Then basically create a wrapper for this package so with the first arg of the CLI you can invoke a method from this file and the remaining args are the args for that function. 

I was just thinking it would be a quick and easy way to easily validate the functionality of the `utils` package.